llvm_util/llvm2alive.cpp

```diff
@@ -694,13 +694,40 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
       }
     }
 
+    auto getGlobalVariable = [this](const string &name) -> llvm::GlobalVariable * {
+      auto M = f.getParent();
+      // If name is a numeric value, the result should be manually found
+      bool is_numeric = name.size() == 1 && name[0] == '0';
+      unsigned numeric_id = 0;
+      if (!is_numeric) {
+        is_numeric = true;
```

Comment:
it seems that is_numeric is always true after this. the code seems funny.
Also, please use something like strtoul() instead of doing it by hand